### ISSUE TITLE
cnn using variable length data no padding zeros

### ISSUE THREAD

**Issue by wjbianjason** [fchollet/keras#701]
I have read issue #424 , I also need to use cnn dealing with varialble length input,I want to know is there any method that is no need to pad zeros or the theory that padding zeros is no influence for the result.I don't want to change the different kind of neural network btw. Thanks for  your answer.

**Comment by suixudongi8** [comment]
Change the border_mode param.
There are two options:
{'same', 'valid'}

**Comment by McVilla** [comment]
@wjbianjason , I have the same use case. Did you find any answers finally?
